docs/devguide/modules/install-sbo/partials/minikube/olm-install.adoc

```diff
@@ -0,0 +1,34 @@
+== Install with OLM
+
+1. Enable OLM
+[source,bash]
++
+----
+minikube addons enable olm
+----
+
+2. Configure the shell
++
+[source,bash]
+----
+export OPERATOR_REPO_REF=$(minikube ip):5000/service-binding-operator
+export OPERATOR_INDEX_IMAGE_REF=$(minikube ip):5000/service-binding-operator:index
+export OPERATOR_BUNDLE_IMAGE_REF=$(minikube ip):5000/sbo-bundle:latest
+----
+
+3. Build and push images
++
+[source,bash]
+----
+make image push-image -o registry-login
+make bundle-image push-bundle-image -o registry-login
+make index-image push-index-image -o registry-login OPM_RENDER_OPTS=--use-http
+----
+
+4. Deploy the operator through with OLM
++
+[source,bash]
+----
+make deploy-from-index-image
```

Comment:
It would be also usefull to show how to deploy SBO from a given index image. For example:

* From operatorhub.io:
```
CATSRC_NAME=operatorhubio OPERATOR_INDEX_IMAGE_REF=quay.io/operatorhubio/catalog:latest make deploy-from-index-image
```
* For installing SBO for a given PR (`PR_NUMBER`) and the PR's `COMMIT_ID`, which is first 8 characters of the PR's commit ID:
```
CATSRC_NAME=sbo-pr OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:pr-<PR_NUMBER>-<COMMIT_ID>-index make deploy-from-index-image
```